models/oneinch/ethereum/oneinch_v1_ethereum_trades.sql

```diff
@@ -30,218 +31,218 @@
     ,'contract_address'
 ] %}
 
-WITH oneinch_calls AS
-(
+WITH
+  oneinch_calls AS (
     SELECT
-        call_block_number as block_number,
-        CAST(NULL as string) as taker,
-        fromToken as from_token,
-        toToken as to_token,
-        tokensAmount as from_amount,
-        minTokensAmount as to_amount,
-        call_tx_hash as tx_hash,
-        call_block_time as block_time,
-        call_trace_address as trace_address,
-        CAST(-1 as integer) AS evt_index,
-        contract_address
+      call_block_number AS block_number,
+      CAST(NULL AS VARBINARY) AS taker,
+      fromToken AS from_token,
+      toToken AS to_token,
+      tokensAmount AS from_amount,
+      minTokensAmount AS to_amount,
+      call_tx_hash AS tx_hash,
+      call_block_time AS block_time,
+      call_trace_address AS trace_address,
+      CAST(-1 AS INTEGER) AS evt_index,
```

Comment:
![image](https://github.com/duneanalytics/spellbook/assets/102681548/b61d57a7-d1f4-485e-b2fb-daa874b70516)
changing to `bigint` based on this

Reply:
chose this table since it's in `v2` oneinch spell that will union with this one, should ensure same datatypes